RCAIDE/Framework/Missions/Conditions/Mass.py

```diff
@@ -0,0 +1,111 @@
+# RCAIDE/Framework/Missions/Conditions/Mass.py
+# (c) Copyright 2024 Aerospace Research Community LLC
+#
+# Created: Jul 2024, RCAIDE Team
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------
+
+import unittest
+from dataclasses import dataclass, field
+
+# package imports
+import numpy as np
+
+# RCAIDE imports
+from RCAIDE.Framework.Missions.Conditions import Conditions
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  Mass
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+@dataclass(kw_only=True)
+class MassConditions(Conditions):
+    """
+    Represents the mass conditions for a vehicle or system.
+
+    This class extends the Conditions base class to specifically handle mass-related
+    parameters and calculations.
+
+    Attributes
+    ----------
+    name : str
+        The name of the mass conditions.
+
+    total : np.ndarray
+        The total mass of the system.
+    rate_of_change : np.ndarray
+        The rate of change of mass.
+
+    total_moment_of_inertia : np.ndarray
+        The total moment of inertia.
+
+    breakdown : Conditions
+        A nested Conditions object representing the breakdown of mass components.
+
+    Notes
+    -----
+    All attributes are initialized using default factories to ensure each instance
+    has its own copy of mutable objects.
+    """
+
+    name:                       str         = 'Mass Conditions'
```

Comment:
We need to make this more consistent with components such that we can have variable mass components and fixed mass components and have the weights, cg's, and inertias tracked in the conditions.

Then we can do a quick update calc which takes the fixed weight/cg/inertias and add them together

Reply:
Made the attribute names match the `MassProperties` class attributes. Future mission solver PR will incorporate symmetry into mass updating function.